plugin/pkg/admission/resourcequota/admission_test.go

```diff
@@ -796,35 +724,24 @@ func TestAdmitPodInNamespaceWithoutQuota(t *testing.T) {
 			},
 		},
 	}
-	liveLookupCache, err := lru.New(100)
-	if err != nil {
-		t.Fatal(err)
-	}
+
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 
 	kubeClient := fake.NewSimpleClientset(resourceQuota)
-	informerFactory := informers.NewSharedInformerFactory(kubeClient, controller.NoResyncPeriodFunc())
-	quotaAccessor, _ := newQuotaAccessor()
-	quotaAccessor.client = kubeClient
-	quotaAccessor.lister = informerFactory.Core().V1().ResourceQuotas().Lister()
-	quotaAccessor.liveLookupCache = liveLookupCache
-	config := &resourcequotaapi.Configuration{}
-	quotaConfiguration := install.NewQuotaConfigurationForAdmission()
-	evaluator := NewQuotaEvaluator(quotaAccessor, quotaConfiguration.IgnoredResources(), generic.NewRegistry(quotaConfiguration.Evaluators()), nil, config, 5, stopCh)
+	informerFactory := informers.NewSharedInformerFactory(kubeClient, 0)
 
-	handler := &QuotaAdmission{
-		Handler:   admission.NewHandler(admission.Create, admission.Update),
-		evaluator: evaluator,
+	handler, err := createHandler(kubeClient, informerFactory, stopCh)
+	if err != nil {
+		t.Errorf("Error occurred while creating admission plugin: %v", err)
 	}
+
 	// Add to the index
 	informerFactory.Core().V1().ResourceQuotas().Informer().GetIndexer().Add(resourceQuota)
-	newPod := validPod("not-allowed-pod", 1, getResourceRequirements(getResourceList("100m", "2Gi"), getResourceList("200m", "")))
-	// Add to the lru cache so we do not do a live client lookup
-	liveLookupCache.Add(newPod.Namespace, liveLookupEntry{expiry: time.Now().Add(time.Duration(30 * time.Second)), items: []*corev1.ResourceQuota{}})
+	newPod := validPod("allowed-pod", 1, getResourceRequirements(getResourceList("100m", "2Gi"), getResourceList("200m", "")))
 	err = handler.Validate(context.TODO(), admission.NewAttributesRecord(newPod, nil, api.Kind("Pod").WithVersion("version"), newPod.Namespace, newPod.Name, corev1.Resource("pods").WithVersion("version"), "", admission.Create, &metav1.CreateOptions{}, false, nil), nil)
 	if err != nil {
```

Comment:
so is the newPod allowed or not-allowed?

Reply:
It's allowed, since the test is called `TestAdmitPodInNamespaceWithoutQuota` and that's what the implementation looks like. Imo, it was a copy paste issue previously.